### PR TITLE
A caseworker can mark an initial assessment as delivered

### PR DESCRIPTION
**Pull request by lawrence-forooghian**
# What's new

A caseworker can now mark an initial assessment as delivered. They are unable to start working on the action plan until the initial assessment is delivered.

# Screenshots

## Marking an initial assessment as delivered
image

## The resultant state
image

### CODE REVIEW

**Comment by lawrence-forooghian** [comment]
[Build the ability to mark initial assessment as delivered](https://trello.com/c/ReIkBDax/402-build-the-ability-to-mark-initial-assessment-as-delivered)

File: app/views/sprint-5/book-and-manage/manage-a-referral/caseworker/includes/intervention/initial-assessment.html [hunk 1]
```diff
@@ -9,14 +9,39 @@
...
+        <div class="govuk-checkboxes__item">
+          <input class="govuk-checkboxes__input" id="initial-assessment-delivered" name="delivered" type="checkbox" value="yes">
+          <label class="govuk-label govuk-checkboxes__label" for="initial-assessment-delivered">
+            Initial assessment has been completed
+          </label>
+        </div>
+      </div>
+    </div>
+
+    <button class="govuk-button">Save initial assessment</button>
```
Comment (Gweaton) [inline app/views/sprint-5/book-and-manage/manage-a-referral/caseworker/includes/intervention/initial-assessment.html:49]: Could "Update" be more appropriate than "Save" here? Not a blocker, feel free to merge if you disagree

File: app/routes/sprint-5/caseworkerManageRoutes.js [hunk 2]
```diff
@@ -437,6 +437,23 @@ router.post("/referrals/:referralIndex/interventions/:interventionIndex/initial-
     res.redirect(`/sprint-5/book-and-manage/manage-a-referral/caseworker/referrals/${req.params.referralIndex}/interventions/${req.params.interventionIndex}`);
 });
 
+router.post("/referrals/:referralIndex/interventions/:interventionIndex/initial-assessment-delivered", (req, res) => {
+    const intervention = findIntervention(req);
+
+    // Because of whatever magic auto-store-data.js is doing, we get "yes" if
+    // JS disabled, and ["yes"] if JS enabled 🤷
```
Comment (Gweaton) [inline app/routes/sprint-5/caseworkerManageRoutes.js:444]: Wow.

**Review by Gweaton — APPROVED** [review]

File: app/views/sprint-5/book-and-manage/manage-a-referral/caseworker/includes/intervention/initial-assessment.html (hunk 1 above)
Comment (lawrence-forooghian) [reply app/views/sprint-5/book-and-manage/manage-a-referral/caseworker/includes/intervention/initial-assessment.html]: Changed it, thanks 👍

**Pull request merged**